docs/source/tutorials/plots_intro.ipynb

```diff
@@ -530,6 +530,7 @@
    "id": "5211f6de-d447-46a8-8652-3c9fd77d7915",
```

Comment:
<p>Piling on to commentary about <code>plot_kwargs</code> -- I would not expect the keys here to be graphical elements. Typically, a function's <code>_kwargs</code>are forwarded as-is somewhere, but this is giving finer control than that. I think it requires some further clarification on this point. Because it's breaking a typical pattern, I would even consider copying the docstring into this notebook cell, rather than just linking to it, so you can really break down what is going on here.</p><p> </p><p>Also you need an oxford comma here too : )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/261/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>